modules/nw-dns-operatorloglevel.adoc

```diff
@@ -5,7 +5,7 @@
 [id="nw-dns-operatorloglevel_{context}"]
 = Setting the CoreDNS Operator log level
 
-Cluster administrators can configure the Operator log level to more quickly track down OpenShift DNS issues. The valid values for `operatorLogLevel` are `Normal`, `Debug`, and `Trace`. `Trace` has the most detailed information. The default `operatorlogLevel` is `Normal`. There are seven logging levels for issues: Trace, Debug, Info, Warning, Error, Fatal and Panic. After the logging level is set, log entries with that severity or anything above it will be logged.
+Log levels for CoreDNS and CoreDNS Operator are set in different manners. Cluster administrators can configure the Operator log level to more quickly track down OpenShift DNS issues. The valid values for `operatorLogLevel` are `Normal`, `Debug`, and `Trace`. `Trace` has the most detailed information. The default `operatorlogLevel` is `Normal`. There are seven logging levels for Operator issues: Trace, Debug, Info, Warning, Error, Fatal, and Panic. After the logging level is set, log entries with that severity or anything above it will be logged.
```

Comment:
For some reason "in different manners" seems awkward - maybe "by using different methods"? It might just be me, so ignore if so. :)